css/index.css

```diff
@@ -172,4 +172,38 @@ header .logo {
 footer {
     text-align: center;
     margin: 40px 0 20px;
+}
+@media(max-width:768px){
+    .container{
+        width: 100%;
+        padding-right:20px;
+        padding-left: 20px;
+    }
+    header{
+        width:100%;
+        display:flex;
+        flex-direction: column-reverse;
+        align-items: center; 
+    }
+    header nav{
+        width:100%;
+    }
+    header .logo{
```

Comment:
Almost, keep at it. Just mind the fact that its sliding with the scroll. I'm sure one of the class mates can help you get this.